docs/security/programmatically-lock-user-accounts.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Programmatically lock and unlock user accounts
+description: Lock users to prevent them from signing in based on your own custom criteria.
+---
+
+# Programmatically lock and unlock user accounts
+
+## Unlocking a user programmatically
+
+You can programmatically unlock a user using the [UnlockUser](https://clerk.com/docs/reference/backend-api/tag/Users#operation/LockUser) Backend API operation. 
+
+## Usecases
+
+### "Unlock button"
```

Comment:
Just realized all of these headings are out of order. 
So it should be
## Unlocking a user programmatically
### Use cases
#### Unlock button
#### Send an unlock request to an admin